azure-pipelines.yml

```diff
@@ -9,14 +9,30 @@ steps:
 - task: Gradle@2
   inputs:
     gradleWrapperFile: 'gradlew'
-    tasks: 'build'
+    tasks: 'assembleRelease makeUniversalPkg'
+    publishJUnitResults: true
+    testResultsFiles: '**/TEST-*.xml'
     javaHomeOption: 'JDKVersion'
     sonarQubeRunAnalysis: false
+    options: '-PappCenterSecret=$(appCenterSecret) -PsigningKeyPassword=$(signingKeyPassword) -PsigningKeyStorePassword=$(signingKeyStorePassword)'
```

Comment:
@aishwarya56  can you verify options here

Reply:
The options here seems fine. Do we need maven options too?
Also, figure out a way to create a different azure pipeline. If in future we had to merge it in master this will create conflict for main pipeline.

Reply:
Maven options will be needed after final modularization implementation is complete. For now its not required.
I will figure out a way to create different build pipeline as well after modularization. This unblocks the current integration for now